### DISCUSSION TITLE
Carousel data-pause on each slide

### DISCUSSION THREAD

**Discussion by crustamet** [twbs/bootstrap#30763] [General]
I have a carousel with first slide with a video and the rest they are just images

I want the first slide to disable pause on hover while the rest slides keeps data-pause on hover.

Do you guys think it is possible with current version of bootstrap ?

**Comment by mdo** [comment]
This is more appropriate for a how-to place like our Discussions, our Slack, or Stack Overflow. Converting it to a discussion for now in case anyone in the community has some ideas for you.
